concordium-node/src/stats_export_service.rs

```diff
@@ -467,6 +470,16 @@ impl StatsExportService {
         let last_throughput_measurement_received_bytes = AtomicU64::new(0);
         let avg_bps_in = AtomicU64::new(0);
         let avg_bps_out = AtomicU64::new(0);
+        let recent_peers = IntGaugeVec::new(
+            Opts::new(
+                "recent_peers",
```

Comment:
I think the name, if it was to follow other metrics we have, would be more like `peer_bucket_size`.